charts/hookd/templates/secret.yaml

```diff
@@ -27,3 +27,4 @@ stringData:
   HOOKD_PROVISION_KEY: "{{ .Values.provisionKey }}"
   HOOKD_DATABASE_URL: "postgres://{{ .Values.database.user }}@127.0.0.1:5432/{{ .Values.database.name }}?sslmode=disable"
   HOOKD_FRONTEND_KEYS: "{{ .Values.frontendPreSharedKey }}"
+  HOOKD_TEAMS_API_KEY: "{{ .Values.teamsAPIKey }}"
```

Comment:
This is already mapped under the old name, it seems:
``` 
HOOKD_CONSOLE_URL: "http://teams-backend/query"
HOOKD_CONSOLE_API_KEY: "{{ .Values.consoleAPIKey }}"
```

should probably remove the old ones